.github/workflows/test.yml

```diff
@@ -29,6 +29,8 @@ jobs:
           cache: "pnpm"
       - name: Install Dependencies
         run: pnpm install --frozen-lockfile
+      - name: Test build
+        run: pnpm run build
```

Comment:
Updated the test workflow to run a build to capture any issues before we release.